spec/app_version_spec.rb

```diff
@@ -31,9 +31,13 @@
   end
 
   describe 'the deprecated version' do
-    let(:sample_version) { ZendeskAppsSupport::AppVersion::DEPRECATED.sample }
+    before do
+      stub_const('ZendeskAppsSupport::AppVersion::DEPRECATED', ['1.0'])
```

Comment:
Nit: Not sure if it matters for these tests, but should the deprecated version be different to the "to be served" version?

Reply:
Not necessarily, the `TO_BE_SERVED` is a superset of the `DEPRECATED` version, so they can be the same as well.
As a good practice, tests should have only the minimal data to run, so it's clear and easy to understand.